docs/Model-Notes/README.md

```diff
@@ -13,7 +13,6 @@ Arbor Networks|[ArbOS](ArbOS.md)|27 Feb 2018
 Arista|[EOS](EOS.md)|05 Feb 2018
 Huawei|[VRP](VRP-Huawei.md)|17 Nov 2017
 Juniper|[MX/QFX/EX/SRX/J Series](JunOS.md)|18 Jan 2018
-Netgear|[Netgear](Netgear.md)|11 Apr 2018
```

Comment:
Looks like the rebase has removed the Netgear model. Can you re-add that back in please.